packages/satori-js/package.json

```diff
@@ -31,17 +31,5 @@
         "Andrei Mihu <andrei@heroiclabs.com>",
         "Mo Firouz <mo@heroiclabs.com>"
     ],
-    "license": "Apache-2.0",
-    "dependencies": {
-        "js-base64": "^3.6.0",
-        "whatwg-fetch": "^2.0.3"
-    },
-    "devDependencies": {
-        "@babel/runtime": "^7.12.5",
```

Comment:
Should these be removed for Satori? I think we still want to target all the same platforms including cocos creator.

Reply:
I was also wondering about the other types of Javascript modules we publish (cjs, iife, umd) and if those should be applied to Satori as well.

Reply:
`whatwg-fetch` should go back in yes (as well as `js-base64` and `base64-arraybuffer`, good catch.
`babel` isn't needed as `esbuild` handles transpiling.

Yep `satori-js` builds all the appropriate js module formats (specifically cjs, esm, iife and umd). I do need to add the `rollup -c --bundleAsCjs` back into the `build` script though.